airbyte/helm/airbyte/Chart.lock

```diff
@@ -1,7 +1,7 @@
 dependencies:
 - name: airbyte
-  repository: file://../../../../airbyte/charts/airbyte
-  version: 0.43.22
+  repository: https://airbytehq.github.io/helm-charts
```

Comment:
@michaeljguarino if there are upstream changes in your fork that should be included, I'll just need a hand to incorporate those! Thanks

Reply:
yea my fork has some meaningful changes from last i remember

Reply:
could you revert that for now as we can't merge with it as a dependency (you actually shouldn't need to have my fork to perform this upgrade as the chart is in the repo and should be safe to apply w/

Reply:
yep that's it